server/application/application.go

```diff
@@ -956,7 +956,7 @@ func (s *Server) streamApplicationEvents(
 	}
 
 	logCtx.Info("streaming application events")
-	appEvent, err := getApplicationEventPayload(a, es, desiredManifests)
+	appEvent, err := getApplicationEventPayload(a, es, desiredManifests, s, ctx)
```

Comment:
ctx should always pass as first param

Reply:
no need to pass the `server`, just make this function a method and use it as receiver type